public/static/locales/en/me.json

```diff
@@ -32,6 +32,8 @@
   "moreThanOne": "minimum 1 tree required",
   "registered": "Registered",
   "receivedTrees": "Received Trees",
+  "receivedFrom": "Received from {{name}}",
+  "giftTo": "Gifted to {{name}}",
```

Comment:
line 8 has 

`"giftToGiftee": "Gift to ${gifteeName}",`

Reply:
removed `giftTo`